README.md

```diff
@@ -71,7 +79,10 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://github.com/underscoreanuj"><img src="https://avatars1.githubusercontent.com/u/30765911?v=4" width="100px;" alt="Anuj Singh"/><br /><sub><b>Anuj Singh</b></sub></a><br /><a href="https://github.com/devfolioco/scrroll-in/commits?author=underscoreanuj" title="Code">💻</a></td>
     <td align="center"><a href="http://www.gaberosedesign.com"><img src="https://avatars3.githubusercontent.com/u/7225212?v=4" width="100px;" alt="Gabe"/><br /><sub><b>Gabe</b></sub></a><br /><a href="https://github.com/devfolioco/scrroll-in/commits?author=roseg43" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/Alucard17"><img src="https://avatars1.githubusercontent.com/u/26205172?v=4" width="100px;" alt="alucard17"/><br /><sub><b>alucard17</b></sub></a><br /><a href="https://github.com/devfolioco/scrroll-in/commits?author=alucard17" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/3scava1i3r"><img src="https://avatars2.githubusercontent.com/u/52003051?s=96&v=4" width="100px;" alt="3scava1i3r"/><br /><sub><b>3scava1i3r</b></sub></a><br /><a href="https://github.com/devfolioco/scrroll-in/commits?author=3scava1i3r" title="Code">💻</a></td>
```

Comment:
Please don't add this manually here, this is automatically managed by the all-contributors bot

Reply:
Ok should I correct it then? Like delete the same?